tests/setup.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * This file is to provide the correct setup for the Vue instance.
+ * It can save people time when writing tests, as they won't need to figure out
+ * why some of the errors are showing up due to Vue not having the plugins it needs.
+ */
+import Vue from 'vue'
+import Vue2Filters from 'vue2-filters'
+import Vuelidate from 'vuelidate'
+import Vuetify from 'vuetify'
+
+Vue.use(Vue2Filters) // needed by SbcFeeSummary
+Vue.use(Vuelidate) // needed by BaseAddress
+Vue.use(Vuetify)
```

Comment:
What about router?
There's probably more that could be put in here possibly

Reply:
True Travis but I'm trying to stay as consistent between UIs as possible. The other two don't have the router here! Let me know if I should make this change here and in other ones.